runtime/src/main/java/com/datastax/oss/quarkus/runtime/reactive/QuarkusReactiveCqlSession.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.oss.quarkus.runtime.reactive;
+
+import com.datastax.dse.driver.api.core.cql.reactive.ReactiveRow;
+import com.datastax.dse.driver.api.core.graph.GraphStatement;
+import com.datastax.dse.driver.api.core.graph.reactive.ReactiveGraphNode;
+import com.datastax.oss.driver.api.core.CqlSession;
+import com.datastax.oss.driver.api.core.cql.AsyncResultSet;
+import com.datastax.oss.driver.api.core.cql.PrepareRequest;
+import com.datastax.oss.driver.api.core.cql.PreparedStatement;
+import com.datastax.oss.driver.api.core.cql.SimpleStatement;
+import com.datastax.oss.driver.api.core.cql.Statement;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import io.smallrye.mutiny.Multi;
+import io.smallrye.mutiny.Uni;
+
+public class QuarkusReactiveCqlSession {
+  private CqlSession cqlSession;
+
+  public QuarkusReactiveCqlSession(CqlSession cqlSession) {
+    this.cqlSession = cqlSession;
+  }
+
+  @NonNull
+  Multi<ReactiveRow> executeContinuouslyReactive(@NonNull String query) {
```

Comment:
One of the issues with wrapping `ReactiveResultSet` into something else is that we lose its specific methods. This is a problem for Reactor and RxJava because `Flux` and `Flowable` are concrete classes that cannot be extended. However here I see that `Multi` is an interface. I think you should instead explore the idea of creating an interface `MutinyResultSet extends ReactiveResultSet` and see if it's possible to implement it without too much trouble.

Reply:
Which methods do you mean? On the quarkus docs, they describe the `Multi.createFrom()` as a method for doing this conversion.

Reply:
I've created a wrapper around our `ReactiveResultSet` and `Mutiny`, please take a look:
https://github.com/datastax/cassandra-quarkus/pull/11/files#diff-e8a1647fb214d790e2adcf27fe8ca97bR42
and let me know wdyt. 
If you think that this is a good solution, I'll create a similar class for `ReactiveGraphNode`.

Reply:
It looks feasible and not too ugly imo.